### PR TITLE
[Fix] fix mmpose regression codec sdk export

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

fix https://github.com/open-mmlab/mmdeploy/issues/1677

### CODE REVIEW

**Comment by lvhan028** [comment]
Which models can be tested?

**Comment by irexyc** [comment]
> Which models can be tested?

tested with https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/topdown_regression/coco/td-reg_res50_8xb64-210e_coco-256x192.py

**Comment by RunningLeon** [comment]
@irexyc It seems that there is no regression model in [supported models](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/docs/en/04-supported-codebases/mmpose.md#supported-models). BTW, tested and the sdk result is not aligned with pytorch.
```
python tools/deploy.py \
configs/mmpose/pose-detection_tensorrt_static-256x192.py \
../mmpose/configs/body_2d_keypoint/topdown_regression/coco/td-reg_mobilenetv2_rle-pretrained-8xb64-210e_coco-256x192.py \
https://download.openmmlab.com/mmpose/v1/body_2d_keypoint/topdown_regression/coco/td-reg_mobilenetv2_rle-pretrained-8xb64-210e_coco-256x192-39b73bd5_20220922.pth \
demo/resources/human-pose.jpg \
--work-dir ./work_dir \
--device cuda --dump-info
```

```
python tools/test.py \
configs/mmpose/pose-detection_sdk_static-256x192.py \
../mmpose/configs/body_2d_keypoint/topdown_regression/coco/td-reg_mobilenetv2_rle-pretrained-8xb64-210e_coco-256x192.py \
--model  ./work_dir  --device cuda --work-dir ./work_dir
```
#### results
```
Loading and preparing results...
DONE (t=4.88s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *keypoints*
DONE (t=12.42s).
Accumulating evaluation results...
DONE (t=0.53s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] =  0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets= 20 ] =  0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets= 20 ] =  0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] =  0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] =  0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] =  0.001
 Average Recall     (AR) @[ IoU=0.50      | area=   all | maxDets= 20 ] =  0.002
 Average Recall     (AR) @[ IoU=0.75      | area=   all | maxDets= 20 ] =  0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] =  0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] =  0.002
01/31 04:03:48 - mmengine - INFO - Epoch(test) [104125/104125]  coco/AP: 0.0000  coco/AP .5: 0.0000  coco/AP .75: 0.0000  coco/AP (M): 0.0000  coco/AP (L): 0.0000  coco/AR: 0.0011  coco/AR .5: 0.0019  coco/AR .75: 0.0011  coco/AR (M): 0.0007  coco/AR (L): 0.0016

```

**Comment by irexyc** [comment]
>It seems that there is no regression model in [supported models](https://github.com/open-mmlab/mmdeploy/blob/dev->1.x/docs/en/04-supported-codebases/mmpose.md#supported-models). BTW, tested and the sdk result is not aligned >with pytorch.

The wrong result is because when predict, [`RLEHead`](td-reg_mobilenetv2_rle-pretrained-8xb64-210e_coco-256x192.py) add sigmoid layer at the end.

**Comment by RunningLeon** [comment]
> > Which models can be tested?
> 
> tested with https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/topdown_regression/coco/td-reg_res50_8xb64-210e_coco-256x192.py

This model works fine.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by lvhan028** [comment]
> > It seems that there is no regression model in [supported models](https://github.com/open-mmlab/mmdeploy/blob/dev-%3E1.x/docs/en/04-supported-codebases/mmpose.md#supported-models). BTW, tested and the sdk result is not aligned >with pytorch.
> 
> The wrong result is because when predict, [`RLEHead`](td-reg_mobilenetv2_rle-pretrained-8xb64-210e_coco-256x192.py) add sigmoid layer at the end.

Let's resolve it in another PR

**Review by lvhan028 — APPROVED** [review]
LGTM
